Documentation/extending-form-api.rst

```diff
@@ -177,9 +178,13 @@ list. A Form Element Renderer must implement the ``RendererInterface`` interface
 	         $content .= sprintf('<li>%s</li>', htmlspecialchars($item));
 	      }
 	      $content .= '</ul>';
+	      $content = $this->formRuntime->invokeRenderCallbacks($content, $renderable);
 	      return $content;
 	   }
 	}
 
+.. note::  Don't forget to invoke ``RootRenderableInterface::beforeRendering()`` and ``FormRuntime::invokeRenderCallbacks()``
```

Comment:
Yes, my thoughts too. Besides the callback needs to be invoked at a very specific point in time, AOP might not work for all cases.
I considered Signal/Slot but found it a bit awkward because they usually don't modify their payload. Also we need to activate the render callbacks at runtime (i.e. when in the Neos backend). A hard-coded Slot would introduce the overhead to check for activity in the calback.